layouts/index.html

```diff
@@ -0,0 +1,57 @@
+{{ define "header" }}
+<div class="jumbotron-custom">
```

Comment:
this content should be wrapped in a bootstrap 'container' class at some point, as the margins do not match the rest of the content

Reply:
Done